### PR TITLE
DevWs for Zooz (The Smartest House) containing containing Zooz ZSE41 XS Open Close Sensor

### PR DESCRIPTION
**Pull request by KevinTSH**

### CODE REVIEW

**Comment by KevinTSH** [comment]
The ZSE41 does not have the same capabilities or preferences as the ZSE42 so this DTH can't be combined with the one in PR 74778.

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]
Those 5 configuration options doesn't seem to justify brand new DTH, at least in my opinion. Can you integrate this device with existing https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy ?

**Comment by KevinTSH** [comment]
I've deleted the custom handler and added the device's fingerprint to the handler you recommended.

File: devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy [hunk 1]
```diff
@@ -64,6 +64,7 @@ metadata {
 		fingerprint mfr: "0371", prod: "0202", model: "000C", deviceJoinName: "Aeotec Open/Closed Sensor", mnmn: "SmartThings", vid: "generic-contact-5" //AU //Aeotec Door/Window Sensor 7 Pro
 		fingerprint mfr: "0371", prod: "0002", model: "000B", deviceJoinName: "Aeotec Open/Closed Sensor", mnmn: "SmartThings", vid: "generic-contact-5" //EU //Aeotec Door/Window Sensor 7 zw:Ss2a type:0701 mfr:0371 prod:0002 model:000B ver:1.01 zwv:7.12 lib:03 cc:5E,55,9F,6C sec:86,85,8E,59,72,5A,87,73,80,70,71,84,7A
 		fingerprint mfr: "0371", prod: "0102", model: "000B", deviceJoinName: "Aeotec Open/Closed Sensor", mnmn: "SmartThings", vid: "generic-contact-5" //US //Aeotec Door/Window Sensor 7 zw:Ss2a type:0701 mfr:0371 prod:0102 model:000B ver:1.01 zwv:7.12 lib:03 cc:5E,55,9F,6C sec:86,85,8E,59,72,5A,87,73,80,70,71,84,7A
+		fingerprint mfr: "027A", prod: "7000", model: "E001", deviceJoinName: "Zooz Open/Closed Sensor" //Zooz ZSE41 XS Open Close Sensor, raw description: zw:Ss2a type:0701 mfr:027A prod:7000 model:E001 ver:1.05 zwv:7.13 lib:03 cc:5E,55,9F,6C sec:86,85,8E,59,72,5A,87,73,80,71,30,70,84,7A
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy]: Can you move raw description above the fingerprint?

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy (hunk 1 above)
Comment (KevinTSH) [reply devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy:67]: Based on that other PR, is this still necessary?

Comment (KevinTSH) [reply devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy]: I've made the requested change.

**Review by PKacprowiczS — APPROVED** [review]

**Comment by PKacprowiczS** [comment]
live 15.09

**Pull request merged**